parser.y

```diff
@@ -1027,6 +1027,8 @@ import (
 	RoleSpecList                           "Rolename and auth option list"
 	RowFormat                              "Row format option"
 	RowValue                               "Row value"
+	RowStmt                                "Row constructor statement"
```

Comment:
it is just called "row constructor", not a statement :)

```suggestion
	RowStmt                                "Row constructor"
```

Reply:
fixed